App.tsx fails linter

For some reason each non-empty line in App.tsx begins with a whitespace character. It seems like there's probably a good reason for it, and the file gets overwritten anyway, but it causes some confusion when running the linter. 

## Expected results

After running
`npx react-native init MyApp --template react-native-template-typescript` and running `yarn lint` inside MyApp, I would expect the linter not to find errors. It is only linting the version of App.tsx found in this repo. 

## Observed results

The linter fails because each non-empty line of code begins with a whitespace character. 

### Logs
```
➜  MyApp git:(master) yarn lint
yarn run v1.22.10
$ eslint . --ext .js,.jsx,.ts,.tsx

/Users/tylersargent/Code/MyApp/App.tsx
   11:1   error  Delete `·`  prettier/prettier
   12:1   error  Delete `·`  prettier/prettier
   13:3   error  Delete `·`  prettier/prettier
   14:1   error  Delete `·`  prettier/prettier
   15:3   error  Delete `·`  prettier/prettier
   16:1   error  Delete `·`  prettier/prettier
   17:1   error  Delete `·`  prettier/prettier
   18:3   error  Delete `·`  prettier/prettier
   19:3   error  Delete `·`  prettier/prettier
   20:1   error  Delete `·`  prettier/prettier
   22:1   error  Delete `·`  prettier/prettier
   23:3   error  Delete `·`  prettier/prettier
   24:1   error  Delete `·`  prettier/prettier
   25:3   error  Delete `·`  prettier/prettier
   26:1   error  Delete `·`  prettier/prettier
   27:1   error  Delete `·`  prettier/prettier
   28:1   error  Delete `·`  prettier/prettier
   30:1   error  Delete `·`  prettier/prettier
   31:1   error  Delete `·`  prettier/prettier
   32:1   error  Delete `·`  prettier/prettier
   33:1   error  Delete `·`  prettier/prettier
   34:1   error  Delete `·`  prettier/prettier
   35:5   error  Delete `·`  prettier/prettier
   36:7   error  Delete `·`  prettier/prettier
   37:1   error  Delete `·`  prettier/prettier
   38:11  error  Delete `·`  prettier/prettier
   39:11  error  Delete `·`  prettier/prettier
   40:1   error  Delete `·`  prettier/prettier
   41:11  error  Delete `·`  prettier/prettier
   42:1   error  Delete `·`  prettier/prettier
   43:1   error  Delete `·`  prettier/prettier
   44:7   error  Delete `·`  prettier/prettier
   45:1   error  Delete `·`  prettier/prettier
   46:9   error  Delete `·`  prettier/prettier
   47:1   error  Delete `·`  prettier/prettier
   48:11  error  Delete `·`  prettier/prettier
   49:1   error  Delete `·`  prettier/prettier
   50:11  error  Delete `·`  prettier/prettier
   51:1   error  Delete `·`  prettier/prettier
   52:1   error  Delete `·`  prettier/prettier
   53:7   error  Delete `·`  prettier/prettier
   54:1   error  Delete `·`  prettier/prettier
   55:1   error  Delete `·`  prettier/prettier
   56:1   error  Delete `·`  prettier/prettier
   58:1   error  Delete `·`  prettier/prettier
   59:1   error  Delete `·`  prettier/prettier
   61:3   error  Delete `·`  prettier/prettier
   62:1   error  Delete `·`  prettier/prettier
   63:1   error  Delete `·`  prettier/prettier
   65:3   error  Delete `·`  prettier/prettier
   66:1   error  Delete `·`  prettier/prettier
   67:1   error  Delete `·`  prettier/prettier
   68:1   error  Delete `·`  prettier/prettier
   69:9   error  Delete `·`  prettier/prettier
   70:9   error  Delete `·`  prettier/prettier
   71:1   error  Delete `·`  prettier/prettier
   72:9   error  Delete `·`  prettier/prettier
   73:1   error  Delete `·`  prettier/prettier
   74:1   error  Delete `·`  prettier/prettier
   75:11  error  Delete `·`  prettier/prettier
   76:1   error  Delete `·`  prettier/prettier
   77:1   error  Delete `·`  prettier/prettier
   78:13  error  Delete `·`  prettier/prettier
   79:11  error  Delete `·`  prettier/prettier
   80:1   error  Delete `·`  prettier/prettier
   81:13  error  Delete `·`  prettier/prettier
   82:11  error  Delete `·`  prettier/prettier
   83:1   error  Delete `·`  prettier/prettier
   84:13  error  Delete `·`  prettier/prettier
   85:11  error  Delete `·`  prettier/prettier
   86:1   error  Delete `·`  prettier/prettier
   87:13  error  Delete `·`  prettier/prettier
   88:11  error  Delete `·`  prettier/prettier
   89:1   error  Delete `·`  prettier/prettier
   90:9   error  Delete `·`  prettier/prettier
   91:1   error  Delete `·`  prettier/prettier
   92:1   error  Delete `·`  prettier/prettier
   93:3   error  Delete `·`  prettier/prettier
   94:1   error  Delete `·`  prettier/prettier
   96:1   error  Delete `·`  prettier/prettier
   97:3   error  Delete `·`  prettier/prettier
   98:5   error  Delete `·`  prettier/prettier
   99:1   error  Delete `·`  prettier/prettier
  100:3   error  Delete `·`  prettier/prettier
  101:1   error  Delete `·`  prettier/prettier
  102:1   error  Delete `·`  prettier/prettier
  103:5   error  Delete `·`  prettier/prettier
  104:3   error  Delete `·`  prettier/prettier
  105:1   error  Delete `·`  prettier/prettier
  106:5   error  Delete `·`  prettier/prettier
  107:1   error  Delete `·`  prettier/prettier
  108:1   error  Delete `·`  prettier/prettier
  109:3   error  Delete `·`  prettier/prettier
  110:1   error  Delete `·`  prettier/prettier
  111:1   error  Delete `·`  prettier/prettier
  112:3   error  Delete `·`  prettier/prettier
  113:1   error  Delete `·`  prettier/prettier
  115:1   error  Delete `·`  prettier/prettier
```
## Steps to reproduce
```
npx react-native init MyApp --template react-native-template-typescript
cd MyApp
yarn lint
```

--- a/template/App.tsx
+++ b/template/App.tsx
@@ -8,108 +8,108 @@
  * @format
  */
 
- import React from 'react';
- import {
-   SafeAreaView,
-   ScrollView,
-   StatusBar,
-   StyleSheet,
-   Text,
-   useColorScheme,
-   View,
- } from 'react-native';
+import React from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  useColorScheme,
+  View,
+} from 'react-native';
 
- import {
-   Colors,
-   DebugInstructions,
-   Header,
-   LearnMoreLinks,
-   ReloadInstructions,
- } from 'react-native/Libraries/NewAppScreen';
+import {
+  Colors,
+  DebugInstructions,
+  Header,
+  LearnMoreLinks,
+  ReloadInstructions,
+} from 'react-native/Libraries/NewAppScreen';
 
- const Section: React.FC<{
-   title: string;
- }> = ({children, title}) => {
-   const isDarkMode = useColorScheme() === 'dark';
-   return (
-     <View style={styles.sectionContainer}>
-       <Text
-         style={[
-           styles.sectionTitle,
-           {
-             color: isDarkMode ? Colors.white : Colors.black,
-           },
-         ]}>
-         {title}
-       </Text>
-       <Text
-         style={[
-           styles.sectionDescription,
-           {
-             color: isDarkMode ? Colors.light : Colors.dark,
-           },
-         ]}>
-         {children}
-       </Text>
-     </View>
-   );
- };
+const Section: React.FC<{
+  title: string;
+}> = ({children, title}) => {
+  const isDarkMode = useColorScheme() === 'dark';
+  return (
+    <View style={styles.sectionContainer}>
+      <Text
+        style={[
+          styles.sectionTitle,
+          {
+            color: isDarkMode ? Colors.white : Colors.black,
+          },
+        ]}>
+        {title}
+      </Text>
+      <Text
+        style={[
+          styles.sectionDescription,
+          {
+            color: isDarkMode ? Colors.light : Colors.dark,
+          },
+        ]}>
+        {children}
+      </Text>
+    </View>
+  );
+};
 
- const App = () => {
-   const isDarkMode = useColorScheme() === 'dark';
+const App = () => {
+  const isDarkMode = useColorScheme() === 'dark';
 
-   const backgroundStyle = {
-     backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
-   };
+  const backgroundStyle = {
+    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
+  };
 
-   return (
-     <SafeAreaView style={backgroundStyle}>
-       <StatusBar barStyle={isDarkMode ? 'light-content' : 'dark-content'} />
-       <ScrollView
-         contentInsetAdjustmentBehavior="automatic"
-         style={backgroundStyle}>
-         <Header />
-         <View
-           style={{
-             backgroundColor: isDarkMode ? Colors.black : Colors.white,
-           }}>
-           <Section title="Step One">
-             Edit <Text style={styles.highlight}>App.js</Text> to change this
-             screen and then come back to see your edits.
-           </Section>
-           <Section title="See Your Changes">
-             <ReloadInstructions />
-           </Section>
-           <Section title="Debug">
-             <DebugInstructions />
-           </Section>
-           <Section title="Learn More">
-             Read the docs to discover what to do next:
-           </Section>
-           <LearnMoreLinks />
-         </View>
-       </ScrollView>
-     </SafeAreaView>
-   );
- };
+  return (
+    <SafeAreaView style={backgroundStyle}>
+      <StatusBar barStyle={isDarkMode ? 'light-content' : 'dark-content'} />
+      <ScrollView
+        contentInsetAdjustmentBehavior="automatic"
+        style={backgroundStyle}>
+        <Header />
+        <View
+          style={{
+            backgroundColor: isDarkMode ? Colors.black : Colors.white,
+          }}>
+          <Section title="Step One">
+            Edit <Text style={styles.highlight}>App.js</Text> to change this
+            screen and then come back to see your edits.
+          </Section>
+          <Section title="See Your Changes">
+            <ReloadInstructions />
+          </Section>
+          <Section title="Debug">
+            <DebugInstructions />
+          </Section>
+          <Section title="Learn More">
+            Read the docs to discover what to do next:
+          </Section>
+          <LearnMoreLinks />
+        </View>
+      </ScrollView>
+    </SafeAreaView>
+  );
+};
 
- const styles = StyleSheet.create({
-   sectionContainer: {
-     marginTop: 32,
-     paddingHorizontal: 24,
-   },
-   sectionTitle: {
-     fontSize: 24,
-     fontWeight: '600',
-   },
-   sectionDescription: {
-     marginTop: 8,
-     fontSize: 18,
-     fontWeight: '400',
-   },
-   highlight: {
-     fontWeight: '700',
-   },
- });
+const styles = StyleSheet.create({
+  sectionContainer: {
+    marginTop: 32,
+    paddingHorizontal: 24,
+  },
+  sectionTitle: {
+    fontSize: 24,
+    fontWeight: '600',
+  },
+  sectionDescription: {
+    marginTop: 8,
+    fontSize: 18,
+    fontWeight: '400',
+  },
+  highlight: {
+    fontWeight: '700',
+  },
+});
 
- export default App;
+export default App;

